_includes/flutter/files.md

```diff
@@ -0,0 +1,55 @@
+# Files
+
+There are three different file classes in this SDK:
+- `ParseFileBase` is and abstract class and is the foundation of every file class that can be handled by this SDK.
+- `ParseFile` (former the only file class in the SDK) extends ParseFileBase and is by default used as the file class on every platform but web.
+  This class uses a `File` from `dart:io` for storing the raw file.
+- `ParseWebFile` is the equivalent to ParseFile used at Flutter Web.
+  This class uses an `Uint8List` for storing the raw file.
+
+These classes are used by default to represent files, but you can also build your own class extending ParseFileBase and provide a custom `ParseFileConstructor` similar to the `SubClasses`.
```

Comment:
```suggestion
The classes above are used by default to represent files, but you can also build your own class extending `ParseFileBase` and provide a custom `ParseFileConstructor` similar to the `SubClasses`.
```

- Clear reference, if referring to a list above indicate that by "the classes above" or similar, if referring to a list below use "the following classes" or "these classes ... :" with a colon at the end.
- Is `SubClasses` or `SubClass` an actual class that the developer can use? Or is that just an SDK-internal class. If it's an internal class, then write "...similar to subclasses." If `SubClass` exists but `SubClasses` does not, then write "similar to a `SubClass`".